<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="accountLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLabel" name="pwdLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="accountEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>请输入账号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pwdEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>请输入密码</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>loginButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loginClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loginClick()</slot>
  <slot>quitClick()</slot>
 </slots>
</ui>
